Source/Core/DolphinQt/FIFO/FIFOAnalyzer.cpp

```diff
@@ -146,7 +146,7 @@ void FIFOAnalyzer::UpdateTree()
 
   m_tree_widget->addTopLevelItem(recording_item);
 
-  auto* file = m_fifo_player.GetFile();
+  const auto* file = m_fifo_player.GetFile();
```

Comment:
You can make file itself `const`.
```suggestion
  const auto* const file = m_fifo_player.GetFile();
```

Reply:
What do you mean? It is already `const` because of the first `const` on the row.

Reply:
That's not how constness works. `T*`, `const T*`, `T* const` and `const T* const` all mean different things because the constness can apply to the type or the pointer (or both or neither).

Reply:
Ok, thanks!